narwhals/_arrow/series_dt.py

```diff
@@ -189,3 +191,13 @@ def total_nanoseconds(self) -> ArrowSeries:
         }
         factor = lit(unit_to_nano_factor[self.unit], type=pa.int64())
         return self.with_native(pc.multiply(self.native, factor).cast(pa.int64()))
+
+    def truncate(self, every: str) -> ArrowSeries:
+        number, unit = parse_interval_string(every)
+        unit_dict = unit_to_str()
+        if unit == "q" and number != "1":
+            msg = f"Only 1{unit} is currently supported for PyArrow."
```

Comment:
if we want to leave non-1 units with Q for a follow-up, can we add "for 'q'" to the error message? because it looks like it's supported for non-'q' units